client/components/cards/attachments.js

```diff
@@ -38,4 +67,77 @@ Template.cardAttachmentsPopup.events({
     tpl.find('.js-attach-file').click();
     evt.preventDefault();
   },
+  'click .js-upload-clipboard-image'() {
+    const results = Popup.pasted;
+    if((typeof Blob !== 'undefined') && (results.file instanceof Blob) && results.file) {
+      const card = this;
+      const file = new FS.File( results.file );
+      if(typeof results.file.type === 'string') {
+        const extname = results.file.type.split('/')[1];
+        const filename = 'clipboard.png'.replace('png', extname);
+        file.name( filename );
+      }
+      file.updatedAt(new Date());
+      file.boardId = card.boardId;
+      file.cardId  = card._id;
+      Attachments.insert(file);
+    }
+
+    $('html').pasteImageReader(function(){});
+    Popup.close();
+  },
 });
+
+// ------------------------------------------------------------------------
+// Created by STRd6
```

Comment:
You should move this code in a dedicated file, also I would recommend to keep it unmodified. Adding

```js
/*eslint-disable */
```

at the beginning of the file should disabling Eslint for it.